app/controllers/application_controller.rb

```diff
@@ -1,5 +1,21 @@
 class ApplicationController < ActionController::API
 
+  protected
+
   include RescueFromUnlessLocal
 
+  def current_user_uuid
+    @current_user_uuid ||= begin
+      if Rails.env.development? && ENV['STUBBED_USER_UUID']
```

Comment:
do we want a warning here if the STUBBED_USER_UUID is set, but not in development mode?  could this create confusion?  

another ?  is it a problem that this is memoized? if a user is removed or logs out of accounts while this user_uuid is still in use could this be a problem?

Reply:
Sure re first comment.  I made the change, please check my logic.

Not a problem that it is memoized.  Controller instances are created anew for each request, so this memoized variable only lasts for the duration of one request.  The login state cannot change within a request because it is based on the cookie which doesn't change during the request (at least not to Rails' knowledge)